react-backend/routes/fetchVisitsFilteredBy.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+
+const router = express.Router();
+
+const getVisitsFilteredBy = require('../database/queries/getVisitsFilteredBy');
+
+router.post('/', (req, res, next) => {
+  let body = '';
+  req.on('data', (chunk) => {
```

Comment:
Once the header is set in the fetch above then this can be removed and replaced with req.body